examples/flava/definitions.py

```diff
@@ -74,6 +74,7 @@ class TrainingArguments:
     adam_weight_decay: float = 0.01
     adam_betas: Tuple[float, float] = field(default_factory=lambda: (0.9, 0.999))
     warmup_steps: int = 2000
+    strategy: str = "ddp"
```

Comment:
same as other comment. this should be specifiable through lightning config in line 68